docs/datasets_and_tables/gtfs_rt.md

```diff
@@ -0,0 +1,62 @@
+# GTFS Realtime
+
+Currently, Payments data is hosted by Littlepay, who exposes the "Littlepay Data Model" as a set of files stored in an S3 bucket. To get a copy of the data docs, email hunter.
```

Comment:
Please update from copied payments docs.

Reply:
ah sorry--should be updated now